packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -130,7 +127,6 @@ export class OdspDocumentService implements IDocumentService {
 
         this.hostPolicy = hostPolicy;
         if (this.odspResolvedUrl.summarizer) {
-            this.hostPolicy = cloneDeep(this.hostPolicy);
```

Comment:
Didn't get the purpose of this. Please check if any.

Reply:
We are modifying it on the next line, so clone here is to ensure we did not modify the object that caller has and might continue to use. So maybe clone shallow via

this.hostPolicy = {...this.hostPolicy};

Reply:
hmm, makes sense.